core-blocks/latest-comments/edit.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * WordPress dependencies
+ */
+import { Component, Fragment } from '@wordpress/element';
+import {
+	Disabled,
+	PanelBody,
+	RangeControl,
+	ToggleControl,
+	ServerSideRender,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import {
+	InspectorControls,
+	BlockAlignmentToolbar,
+	BlockControls,
+} from '@wordpress/editor';
+
+/**
+ * Internal dependencies.
+ */
+import './editor.scss';
+
+const MIN_COMMENTS = 1;
+const MAX_COMMENTS = 100;
+
+class LatestComments extends Component {
+	constructor() {
+		super( ...arguments );
+
+		this.setAlignment = this.setAlignment.bind( this );
+		this.setCommentsToShow = this.setCommentsToShow.bind( this );
+		this.toggleAttribute = this.toggleAttribute.bind( this );
+	}
+
+	toggleAttribute( propName ) {
+		return () => {
+			const value = this.props.attributes[ propName ];
+			const { setAttributes } = this.props;
+
+			setAttributes( { [ propName ]: ! value } );
+		};
+	}
+
+	setAlignment( align ) {
+		this.props.setAttributes( { align } );
+	}
+
+	setCommentsToShow( commentsToShow ) {
+		this.props.setAttributes( { commentsToShow } );
+	}
+
+	render() {
+		const {
+			align,
+			commentsToShow,
+			displayAvatar,
+			displayDate,
+			displayExcerpt,
+		} = this.props.attributes;
+
+		return (
+			<Fragment>
+				<BlockControls>
+					<BlockAlignmentToolbar
+						value={ align }
+						onChange={ this.setAlignment }
+					/>
+				</BlockControls>
+				<InspectorControls>
+					<PanelBody title={ __( 'Latest Comments Settings' ) }>
+						<ToggleControl
+							label={ __( 'Display avatar' ) }
+							checked={ displayAvatar }
+							onChange={ this.toggleAttribute( 'displayAvatar' ) }
+						/>
+						<ToggleControl
+							label={ __( 'Display date' ) }
+							checked={ displayDate }
+							onChange={ this.toggleAttribute( 'displayDate' ) }
+						/>
+						<ToggleControl
+							label={ __( 'Display excerpt' ) }
+							checked={ displayExcerpt }
+							onChange={ this.toggleAttribute( 'displayExcerpt' ) }
+						/>
+						<RangeControl
+							label={ __( 'Number of comments' ) }
+							value={ commentsToShow }
+							onChange={ this.setCommentsToShow }
+							min={ MIN_COMMENTS }
+							max={ MAX_COMMENTS }
+						/>
+					</PanelBody>
+				</InspectorControls>
+				<Disabled>
```

Comment:
Makes me wonder: Is there any case where we want `ServerSideRender` to _not_ be disabled? Should this be built-in to the render logic of that component?

Reply:
I wondered that too, we should probably open an issue for it. I suppose building it in and then having some escape hatch prop that *doesn't* disable it would be good.